### PR TITLE
Add luckycat70

### PR DESCRIPTION
**Pull request by rhmokey**
## Description
Add luckycat70

## QMK Pull Request 
https://github.com/qmk/qmk_firmware/pull/22622

## Checklist

- [x] The VIA support for this keyboard is **MERGED** in QMK master already **(MANDATORY)**
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have a V3 JSON version for this keyboard definition.**(MANDATORY)**
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [x] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

### CODE REVIEW

File: v3/mokey/mokey12x2/mokey12x2.json [hunk 1]
```diff
@@ -0,0 +1,125 @@
+{
+  "name": "mokey12x2",
+  "vendorId": "0x6653",
+  "productId": "0x012A",
+  "matrix": { "rows": 4,"cols": 5 },
+  "keycodes": ["qmk_lighting"], 
+  "menus": [
```
Comment (Xelus22) [inline v3/mokey/mokey12x2/mokey12x2.json:7]: https://github.com/qmk/qmk_firmware/tree/master/keyboards/mokey/mokey12x2

There is no RGB stuff here?

File: v3/mokey/luckycat70/luckycat70.json [hunk 2]
```diff
@@ -0,0 +1,150 @@
+{
+  "name": "lucky cat 70",
+  "vendorId": "0x6653",
+  "productId": "0x0070",
+  "matrix": { "rows": 5,"cols": 17 },
+  "keycodes": ["qmk_lighting"], 
+  "menus": ["qmk_rgb_matrix"],
+  "layouts": {
+    "keymap":[
```
Comment (Xelus22) [inline v3/mokey/luckycat70/luckycat70.json:9]: https://www.caniusevia.com/docs/layouts#kle-json-rules

Colour your keys please.

File: v3/mokey/luckycat70/luckycat70.json [hunk 3]
```diff
@@ -0,0 +1,150 @@
+{
+  "name": "lucky cat 70",
+  "vendorId": "0x6653",
+  "productId": "0x0070",
+  "matrix": { "rows": 5,"cols": 17 },
+  "keycodes": ["qmk_lighting"], 
+  "menus": ["qmk_rgb_matrix"],
```
Comment (Xelus22) [inline v3/mokey/luckycat70/luckycat70.json:7]: should be 
```suggestion
  "menus": ["qmk_rgblight"],
```

**Review by Xelus22 — CHANGES_REQUESTED** [review]

File: v3/mokey/mokey12x2/mokey12x2.json (hunk 1 above)
Comment (rhmokey) [reply v3/mokey/mokey12x2/mokey12x2.json:7]: Mokey12x2.json is not being uploaded, but I don't know how to close it. Please help me close it. Thank you

Comment (Xelus22) [reply v3/mokey/mokey12x2/mokey12x2.json:7]: I cannot remove it for you. You must do it yourself

**Pull request closed without merging**